### ISSUE TITLE
Hacktoberfest PRs Welcome

### ISSUE THREAD

**Issue by samuelbeard** [samuelbeard/i-be#1]
Everyone is welcome to add their profile card to this repo.

Once they are published, anyone will be able to see your card using `npx`

Like this:
```
npx i-be samuelbeard
```

**Comment by nikhildev** [comment]
@samuelbeard  Is it ok to create profile creation as an issue and stick it to a PR later?

**Comment by samuelbeard** [comment]
No problem. 😄

**Comment by susanahan** [comment]
@samuelbeard i cant push my changes to make a pr, do i need to have access?

**Comment by samuelbeard** [comment]
@susanahan Hi

Did you fork the repo? You wouldn't be able to make changed to this repo but to the one your fork. Then you can submit a PR from your forked repo to this one.

Just a heads up though. Hacktoberfest doesn't start until Tuesday so if you want your PR to contribute to that, wait until then.

**Comment by susanahan** [comment]
@samuelbeard good point! i didn't know i thought it started as soon as it was open...

and i get this error message 

```
remote: Permission to samuelbeard/i-be.git denied to susanahan.
fatal: unable to access 'https://github.com/samuelbeard/i-be.git/': The requested URL returned error: 403
```

**Comment by samuelbeard** [comment]
@susanahan 

So you are trying to push to this repo and not your own forked version.

Fork this repo and clone your forked version to your machine. Then create a new branch and make changes to that and push them.

When you go back to your forked repo on Github, there should be a button suggesting you submit a PR to this repo.

**Comment by abraaoduarte** [comment]
> Everyone is welcome to add their profile card to this repo.
> 
> Once they are published, anyone will be able to see your card using `npx`
> 
> Like this:
> 
> ```
> npx i-be samuelbeard
> ```

#23 I add my profile
